diff --git a/Lib/site-packages/fps_inspector_sdk/fps_inspector.py b/Lib/site-packages/fps_inspector_sdk/fps_inspector.py
index e47a1c4..480043a 100644
--- a/Lib/site-packages/fps_inspector_sdk/fps_inspector.py
+++ b/Lib/site-packages/fps_inspector_sdk/fps_inspector.py
@@ -119,7 +119,7 @@ def get_all_fliprates ():
     time_arr = numpy.zeros (sample_count).astype (numpy.float64)
     fliprate_arr = numpy.zeros (sample_count*6).astype (numpy.float64)

-    res = PresentMonDLL.get_instance().GetData (current_size, time_arr, fliprate_arr)
+    res = PresentMonDLL.get_instance().GetData (sample_count, time_arr, fliprate_arr)
     if res != PresentMonExitCodes.STATUS_OK.value:
         raise FpsInspectorError ('unable to get fliprate data', res)

